task-list/app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_for @task do |f| %>
```

Comment:
Good use of a partial for this form. However, how does the `form_for` tag know what action it should be doing (creating or updating)?